### ISSUE TITLE
Faraday dependency very old

### ISSUE THREAD

**Issue by cooperia** [keolo/mixpanel_client#42]
When installing the gem, one often hits the following:
```
Bundler could not find compatible versions for gem "faraday":
  In Gemfile:
    mixpanel-client (>= 0) ruby depends on
      faraday (~> 0.8.0) ruby

    twitter (~> 5.7.1) ruby depends on
      faraday (0.9.0)
```

I recommend either bumping up the locked version of typhoeus or removing the version lock entirely.

**Comment by keolo** [comment]
Thanks! I'll fix this asap. I'm also open to pull requests. ;)

**Comment by cooperia** [comment]
Here you go https://github.com/keolo/mixpanel_client/pull/43. =)

One test was failing locally due to very deprecated stuff, is green now.

**Comment by keolo** [comment]
Awesome! I'll merge and release a new gem soon.

**Comment by keolo** [comment]
I merged in your pr and released a new version `4.1.2`. Thanks!
